experiments/misc/ilamb_conversion.jl

```diff
@@ -0,0 +1,389 @@
+        "kg m-2 s-1",
+        "m s^-1",
+    ),
+    "ssr" => ILAMBMapping(
+        "mrros",
+        "Subsurface Runoff",
+        1000.0,
+        "kg m-2 s-1",
+        "m s^-1",
+    ),
+    "swe" =>
+        ILAMBMapping("swe", "Snow Water Equivalent", 1000.0, "kg m-2", "m"),
+    "iwc" => ILAMBMapping(
+        "mrsos",
+        "Soil Moisture (top layer)",
+        1.0,
+        "kg m-2",
+        "kg/m^2",
+    ),
+    "msf" => ILAMBMapping("msf", "Moisture Stress Factor", 1.0, "1", ""),
+    "trans" => ILAMBMapping(
+        "tran",
+        "Transpiration",
+        1000.0,
+        "kg m-2 s-1",
+        "m s^-1",
+    ),
+    "swc" => ILAMBMapping(
+        "swc",
+        "Soil Water Content (top 10cm)",
+        1000.0,
+        "kg m-2",
+        "m^3 m^-3",
+    ),
+    "tsoil" =>
+        ILAMBMapping("tsl", "Soil Temperature (top 10cm)", 1.0, "K", "K"),
+    "precip" => ILAMBMapping(
+        "pr",
+        "Precipitation",
+        -1.0,
+        "kg m-2 s-1",
+        "kg m^-2 s^-1",
+    ),
+    "tair" => ILAMBMapping("tas", "Air Temperature", 1.0, "K", "K"),
+)
+
+"""
+    find_netcdf_files_for_ilamb(output_dir)
+
+Given `output_dir`, a directory with NetCDF files produced by CliMA diagnostics,
+find all the paths of the NetCDF files that can be converted to ILAMB.
+
+This function assumes that NetCDF files that have monthly averages end with
+"_1M_average.nc".
+"""
+function find_netcdf_files_for_ilamb(output_dir)
+    all_filepaths = readdir(output_dir, join = true)
+    filter!(filepath -> endswith(filepath, "_1M_average.nc"), all_filepaths)
+    clima_short_names = keys(ILAMB_VARIABLES)
+    filter!(
+        filepath -> find_short_name(filepath) in clima_short_names,
+        all_filepaths,
+    )
+    return all_filepaths
+end
+
+"""
+    make_ilamb_netcdf_file(filepath, save_dir)
+
+Given a `filepath` to a NetCDF file produced by ClimaDiagnostics, create a
+NetCDF file that is compatible with ILAMB.
+"""
+function make_ilamb_netcdf_file(filepath, save_dir)
+    clima_ds = NCDatasets.NCDataset(filepath)
+    clima_short_name = find_short_name(filepath)
+
+    # Shift dates by one month and add 14 days, since ClimaDiagnostics save
+    # dates at the end of the period instead of at the beginning and ILAMB wants
+    # the days to be in the middle of the months
+    dates = clima_ds["date"] .- Dates.Month(1) .+ Dates.Day(14)
+    start_date = first(dates)
+    end_date = last(dates)
+
+    ilamb_file_name =
+        make_ilamb_file_name(clima_short_name, start_date, end_date)
+    NCDatasets.NCDataset(joinpath(save_dir, ilamb_file_name), "c") do ilamb_ds
+        add_temporal_dimension(ilamb_ds, clima_ds, dates)
+        add_nontemporal_dimensions(ilamb_ds, clima_ds)
+        define_var(ilamb_ds, clima_ds, clima_short_name)
+    end
+    close(clima_ds)
+end
+
+"""
+    add_nontemporal_dimensions(ilamb_ds, clima_ds)
+
+Add the nontemporal dimensions to `ilamb_ds` using `clima_ds`.
```

Comment:
```suggestion
Add the nontemporal dimensions to `ilamb_ds` using `clima_ds`,
where both are open NetCDF files.
```
similar comment for the other NetCDF helper functions